### PR TITLE
feat(detector.ts): Detections without streaming support

### PR DESCRIPTION
**Pull request by brikendr**
Add support for getting detections and framing data without streaming camera using a third party streaming application
on the host machine.

Update compodoc documentation for detector class changes

### CODE REVIEW

File: examples/ga_peoplecount/README.md [hunk 1]
```diff
@@ -23,21 +22,29 @@ const sdk = new HuddlySdk(usbApi);
 async function init() {
   await sdk.init();
 
-  sdk.on('ATTACH', (cameraManager) => {
+  sdk.on('ATTACH', async (cameraManager) => {
+    const autozoomCtl = cameraManager.getAutozoomCtl();
```
Comment (jontore) [inline examples/ga_peoplecount/README.md]: I don't think there is any reason to use C shorthand for the method names ```getAutozoomControl```

File: examples/ga_peoplecount/README.md [hunk 2]
```diff
@@ -23,21 +22,29 @@ const sdk = new HuddlySdk(usbApi);
...
+    // Make sure that autozoom (genius framing is started)
+    await autozoomCtl.init();
```
Comment (jontore) [inline examples/ga_peoplecount/README.md]: In this example you actually want to use the detector right, no need for AZ

File: src/components/autozoomCtl.ts [hunk 3]
```diff
@@ -0,0 +1,215 @@
+import IAutozoomCtl from './../interfaces/IAutozoomCtl';
+import AutozoomCtlOpts from './../interfaces/IAutozoomCtlOpts';
+import IDeviceManager from './../interfaces/iDeviceManager';
+import Api from './api';
+
+export default class AutozoomCtl implements IAutozoomCtl {
```
Comment (jontore) [inline src/components/autozoomCtl.ts]: Nice

File: src/components/device/boxfish.ts [hunk 4]
```diff
@@ -235,7 +238,11 @@ export default class Boxfish extends UvcBaseDevice implements IDeviceManager {
       }
   }
 
-   getDetector(opts?: DetectorOpts): IDetector {
+  getAutozoomCtl(opts: AutozoomCtlOpts): IAutozoomCtl {
```
Comment (jontore) [inline src/components/device/boxfish.ts]: I think a better name would be ```getAutozoomControl```

File: examples/ga_peoplecount/README.md (hunk 2 above)
Comment (brikendr) [reply examples/ga_peoplecount/README.md]: Yes, but just making sure that autozoom is enabled/started before we subscribe for detections. Might lead to confusion if you setup detection listener but autozoom is not started on the camera. That was my thinking.

File: examples/ga_peoplecount/README.md (hunk 1 above)
Comment (brikendr) [reply examples/ga_peoplecount/README.md]: Yeah, we can do that.

**Pull request merged**